frontend/src/pages/Login.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { Wrapper } from 'lib'
```

Comment:
Spotted another missing semi-colon here. And ./ missing

Reply:
Fixed!